README.md

```diff
@@ -0,0 +1,195 @@
+down (pic below) and then configure just like [Configure a job to send
+files to a windows
+share](http://localhost:8085/display/JENKINS/Publish+Over+CIFS+Plugin#PublishOverCIFSPlugin-configjob)
+below
+
+![cifs_builder](docs/images/cifs_builder.png)
+
+#### Configure a job to send files to a windows share
+
+Open a jobs main page and then click "Configure" from the left hand
+menu.
+
+Find the "Send build artifacts to a windows share" checkbox in the
+"Post-build Actions" section and click in the box.
+
+![cifs_unconfigured](docs/images/cifs_unconfigured.png)
+
+You should now have the configuration options as below
+
+![cifs_configured](docs/images/cifs_configured.png)
+
+[see Publish Over ... for common options for
+Server](http://localhost:8085/display/JENKINS/Publish+Over#PublishOver-server)
+
+[see Publish Over ... for common options for Transfer
+Sets](http://localhost:8085/display/JENKINS/Publish+Over#PublishOver-transfer)
+
+Source files
+
+The build will fail if you do not select any Source files to transfer
+
+##### Advanced (Transfer Sets)
+
+If you click the "Advanced..." button for a Transfer Set, then you will
+make more configuration options available (see below)  
+![cifs_advanced](docs/images/cifs_advanced.png)
+
+[see Publish Over ... for common options for Transfer
+Sets](http://localhost:8085/display/JENKINS/Publish+Over#PublishOver-transfer)
+
+###### Clean remote
+
+Delete all files and directories from the remote directory before
+uploading the new files.
+
+##### Advanced (Publisher)
+
+If you click the "Advanced..." button that is immediately below the "Add
+Server" button, then you will make more configuration options available
+(see below)  
+![cifs_publisher_advanced](docs/images/cifs_publisher_advanced.png)
+
+[see Publish Over ... for common options for
+Publisher](http://localhost:8085/display/JENKINS/Publish+Over#PublishOver-publisher)
+
+#### Options to override the default Promotion behaviour
+
+If you are configuring an action in a promotion, new options will become
+available.  
+![cifs_promotions](docs/images/cifs_promotions.png)
+
+[see Publish Over ... for common options for
+Promotions](http://localhost:8085/display/JENKINS/Publish+Over#PublishOver-promotions)
+
+# Change log
+
+#### 0.11 (2018-07-30)
+
+-   [Fix security
+    issue](https://jenkins.io/security/advisory/2018-07-30/#SECURITY-975)
+
+#### 0.9
+
+-   Fixed issue with parameter verification using incorrect check urls
+-   Fixed issue with bufferSize not being set correctly on ugprade
+    ([issue
+    49010](https://issues.jenkins-ci.org/browse/JENKINS-49010))  
+      
+
+Questions, Comments, Bugs and Feature Requests
+
+Please post questions or comments about this plugin to the [Jenkins User
+mailing list](http://jenkins-ci.org/content/mailing-lists).  
+To report a bug or request an enhancement to this plugin please [create
+a ticket in
+JIRA](https://issues.jenkins-ci.org/browse/JENKINS/component/15850).
```

Comment:
Use the same entry page as is used by the plugins site "Report an issue" link.  Reduces errors by those reporting an issue and is less intimidating than jumping directly into Jira.
```suggestion
JIRA](https://www.jenkins.io/participate/report-issue/redirect/#15850).
```